src/s390x/zipl.rs

```diff
@@ -86,8 +233,32 @@ pub fn zipl<P: AsRef<Path>>(boot: P) -> Result<()> {
         boot.join("loader/entries")
     };
 
-    runcmd!("zipl", "--blsdir", blsdir, "--config", conffile.path())?;
-
+    if secure_execution_is_enabled()? {
```

Comment:
> coreos-installer install --append-karg foobar still works in SE mode, right?

Not really, it'd be only possible to have SE with `qemu-secex` image

Reply:
Ahh right, good point. That made we realize something related to this: `secure_execution_is_enabled()` will return `true` even in the install case if the user happens to be running `coreos-installer install` from a Secure Execution-enabled host.

Maybe let's add an argument to this `zipl` function so that we only do the Secure Execution check in the "in-host" case (i.e. from `rdcore zipl`)?